src/components/List/HierarchyList/HierarchyList.jsx

```diff
@@ -264,39 +264,41 @@ const HierarchyList = ({
   const delayedSearch = useCallback(debounce(textInput => handleSearch(textInput), 150), [items]);
 
   return (
-    <List
-      title={title}
-      buttons={buttons}
-      search={
-        hasSearch
-          ? {
-              value: searchValue,
-              onChange: evt => {
-                setSearchValue(evt.target.value);
-                delayedSearch(evt.target.value);
-              },
-            }
-          : null
-      }
-      items={pageSize != null ? itemsToShow : filteredItems}
-      expandedIds={expandedIds}
-      toggleExpansion={id => {
-        if (expandedIds.filter(rowId => rowId === id).length > 0) {
-          // remove id from array
-          setExpandedIds(expandedIds.filter(rowId => rowId !== id));
-        } else {
-          setExpandedIds(expandedIds.concat([id]));
+    <div data-floating-menu-container style={{ position: 'relative' }}>
```

Comment:
Make sure you add the comment and carbon issue link here so others know why its here

Reply:
I have the addons issue linked in the PR description already. I will add the carbon issue to the description.

Reply:
This doesn't seem to be an issue for `SimpleList`. https://carbon-addons-iot-react.com/?path=/story/watson-iot-experimental-simplelist--list-with-overflow-menu

Reply:
At the top of the PR description, set this PR to close that addons issue.

I meant to add a code comment above that `data-floating-menu-container` so developers will understand why its there

Reply:
Yes. I added it. Sorry, I miss-read.